binary_search/binarysearch.cpp

```diff
@@ -0,0 +1,38 @@
+#include<iostream>
+using namespace std;
+int binarySearchIterative(int array[], int start, int end, int find)
```

Comment:
Use underscores for naming variables and functions instead of `camelCase` in whole code.